src/test/unit_tests/test_namespace_fs.js

```diff
@@ -574,11 +591,17 @@ mocha.describe('namespace_fs', function() {
         });
 
         mocha.after(async function() {
-            const delete_res = await ns_tmp.delete_object({
+            let delete_res = await ns_tmp.delete_object({
                 bucket: upload_bkt,
                 key: upload_key_2,
             }, dummy_object_sdk);
             console.log('delete_object with trailing / (key 2) response', inspect(delete_res));
+
+            delete_res = await ns_tmp.delete_object({
+                bucket: upload_bkt,
+                key: upload_key_empty,
+            }, dummy_object_sdk);
+            console.log('delete_object with trailing / (empty dir) response', inspect(delete_res));
```

Comment:
Is this printing updated?  the directory is empty but has body size of 0.

Reply:
I guess it prints the delete_object response. it should contain the version-id, and delete-marker. it isn't related to object size. not sure why we need it. but we do it for the previous object. so might as well do it here too

Reply:
I meant only for the text - I thought you would like to mention it is empty content dir (no just empty dir).